### PR TITLE
[doc] PR for Mention port 18018 requirement for 2.16 and so on in YB docs

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-6740

Note to reviewers: I didn't mention the specific releases in the doc changes because only the affected doc versions were modified and will be published.

### CODE REVIEW

File: docs/content/preview/reference/configuration/default-ports.md [hunk 1]
```diff
@@ -34,9 +34,11 @@ Internode (server-to-server or node-to-node) communication, including xCluster,
 
 To enable login to the machines running these servers, the SSH port 22 should be opened.
 
-xCluster uses the YB-Master port 7100 for the initial communication, and then uses the YB-TServer port 9100 to get data changes.
+xCluster uses the YB-Master port 7100 for the initial communication, and then uses the YB-TServer port 9100 to get data changes. 
 
-Note that YugabyteDB Anywhere obtained the replication lag using Prometheus metrics from tserver (port 9000). So, although the xCluster replication will work if this port is closed, the replication lag cannot be shown on YugabyteDB Anywhere.
+Note that YugabyteDB Anywhere obtains the replication lag information using Prometheus metrics from YB-TServer at port 9000. If this port is closed, the xCluster replication is not affected, but YugabyteDB Anywhere would not be able to display the replication lag.
+
+Before installing YugabyteDB Anywhere, the YB Controller port 18018 must be open. Before upgrading the YugabyteDB software, this port must be open on all YugabyteDB nodes.
```
Comment (kkg-yb) [inline docs/content/preview/reference/configuration/default-ports.md:41]: We don't need the 18018 port to be open on the YBA node. We need the port to be open only on the db nodes.

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/on-premises.md [hunk 2]
```diff
@@ -65,15 +65,16 @@ A YugabyteDB Anywhere node has the following requirements:
...
   * 80 – HTTP access to the YugabyteDB Anywhere UI.
-  * 443 - HTTPs access to the YugabyteDB Anywhere UI.
-  * 22 – SSH
-  * 9090 - Prometheus metrics
-
-  For more information on ports used by YugabyteDB, refer to [Default ports](../../../../reference/configuration/default-ports).
-
+  * 443 – HTTPs access to the YugabyteDB Anywhere UI.
+  * 22 – SSH.
+  * 18018 – Upgrades via YB Controller.
```
Comment (kkg-yb) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/on-premises.md:75]: We don't need the 18018 port to be open on the YBA nodes.

File: docs/content/stable/reference/configuration/default-ports.md [hunk 3]
```diff
@@ -34,7 +34,9 @@ To enable login to the machines running these servers, the SSH port 22 should be
 
 xCluster uses the YB-Master port 7100 for the initial communication, and then uses the YB-TServer port 9100 to get data changes.
 
-Note that YugabyteDB Anywhere obtained the replication lag using Prometheus metrics from tserver (port 9000). So, although the xCluster replication will work if this port is closed, the replication lag cannot be shown on YugabyteDB Anywhere.
+Note that YugabyteDB Anywhere obtains the replication lag information using Prometheus metrics from YB-TServer at port 9000. If this port is closed, the xCluster replication is not affected, but YugabyteDB Anywhere would not be able to display the replication lag.
+
+Before installing YugabyteDB Anywhere, the YB Controller port 18018 must be open. Before upgrading the YugabyteDB software, this port must be open on all YugabyteDB nodes.
```
Comment (kkg-yb) [inline docs/content/stable/reference/configuration/default-ports.md:39]: WE only need the port 18018 top be open on the YugabyteDB nodes and that port should be reachable from the YBA nodes.

**Review by kkg-yb — COMMENTED** [review]
Please note that we do not need the port 18018 to be ope n on the YBA nodes(s). It is only needed to be open in the YugabyteDB nodes. OF course the YBA should be able to access the 18018 port on the DB nodes.

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md [hunk 4]
```diff
@@ -98,7 +98,7 @@ For more information, see [Creating and managing service accounts](https://cloud
 In order to access YugabyteDB Anywhere from outside the GCP environment, you have to enable firewall rules. At a minimum, you need to be able to do the following:
 
 - Access the YugabyteDB Anywhere instance over SSH (port `tcp:22`).
-- Check, manage, and upgrade YugabyteDB Anywhere (port `tcp:8800`).
+- Check, manage, and upgrade YugabyteDB Anywhere (port `tcp:8800` and `tcp:18018`).
```
Comment (vpatibandla-yb) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md:101]: 18018 is not needed for YB Anywhere

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md [hunk 5]
```diff
@@ -109,7 +109,7 @@ Next, you need to create a firewall entry, as follows:
 - Create firewall rules by following instructions provided in [Using firewall rules](https://cloud.google.com/vpc/docs/using-firewalls) in the GCP documentation. When creating the rules:
   - Add a tag `yugabyte-server` to the **Target tags** field.
   - Add the appropriate IP addresses to the **Source IP ranges** field.
-  - Enter a comma-delimited list of TCP ports 22, 8800, 80 to the **Protocol and ports** field. If required, also add TCP ports for a self-managed configuration.
+  - Enter a comma-delimited list of TCP ports 22, 8800, 80, and 18018 to the **Protocol and ports** field. If required, also add TCP ports for a self-managed configuration.
```
Comment (vpatibandla-yb) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md:112]: 18018 for YB Anywhere is not needed

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md [hunk 6]
```diff
@@ -98,7 +98,7 @@ For more information, see [Creating and managing service accounts](https://cloud
 In order to access YugabyteDB Anywhere from outside the GCP environment, you have to enable firewall rules. At a minimum, you need to be able to do the following:
 
 - Access the YugabyteDB Anywhere instance over SSH (port `tcp:22`).
-- Check, manage, and upgrade YugabyteDB Anywhere (port `tcp:8800`).
+- Check, manage, and upgrade YugabyteDB Anywhere (port `tcp:8800` and `tcp:18018`).
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md:101]: tcp:18018 is not needed

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md [hunk 7]
```diff
@@ -62,7 +62,7 @@ type: docs
 In order to access YugabyteDB Anywhere from outside the AWS environment, you would need to enable access by assigning an appropriate security group to the server hosting YugabyteDB Anywhere. At a minimum, you need to be able to do the following:
 
 - Access the YugabyteDB Anywhere instance over SSH (port `tcp:22`).
-- Check, manage, and upgrade YugabyteDB Anywhere (port `tcp:8800`).
+- Check, manage, and upgrade YugabyteDB Anywhere (port `tcp:8800` and `tcp:18018`).
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:65]: 18018 is not needed here

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md [hunk 8]
```diff
@@ -75,7 +75,7 @@ To create a security group that enables these, use the Amazon console to navigat
 
 - Add the appropriate IP addresses to the **Source** field. To allow access from any computer, add `0.0.0.0/0` but note that this is not very secure.
 
-- Add the ports 22, 8800, and 80 to the **Port Range** field. The **Protocol** selected must be TCP.
+- Add the ports 22, 8800, 80, and 18018 to the **Port Range** field. The **Protocol** selected must be TCP.
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:78]: 18018 is not needed here

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/azure.md [hunk 9]
```diff
@@ -77,9 +77,9 @@ You may choose not to create a new security group and instead use an existing on
 
 To access the YugabyteDB Anywhere from outside the Azure environment, you would need to enable access by assigning an appropriate network security group to the YugabyteDB Anywhere machine. At a minimum, you need to be able to do the following:
 
-* Access the YugabyteDB Anywhere instance over SSH (port tcp:22).
-* Check, manage, and upgrade YugabyteDB Anywhere (port tcp:8800).
-* View the YugabyteDB Anywhere UI (port tcp:80).
+* Access the YugabyteDB Anywhere instance over SSH (port `tcp:22`).
+* Check, manage, and upgrade YugabyteDB Anywhere (port `tcp:8800` and `tcp:18018`).
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/azure.md:81]: 18018 is not needed here

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md [hunk 10]
```diff
@@ -62,7 +62,7 @@ type: docs
 In order to access YugabyteDB Anywhere from outside the AWS environment, you would need to enable access by assigning an appropriate security group to the server hosting YugabyteDB Anywhere. At a minimum, you need to be able to do the following:
 
 - Access the YugabyteDB Anywhere instance over SSH (port `tcp:22`).
-- Check, manage, and upgrade YugabyteDB Anywhere (port `tcp:8800`).
+- Check, manage, and upgrade YugabyteDB Anywhere (port `tcp:8800` and `tcp:18018`).
 - View the YugabyteDB Anywhere UI (port `tcp:80`).
 
 If you are using your own Virtual Private Cloud (VPC) as a self-managed configuration, the following additional TCP ports must be accessible: 7000, 7100, 9000, 9100, 11000, 12000, 13000, 9300, 9042, 5433, 6379, 54422. For more information on ports used by YugabyteDB, refer to [Default ports](../../../../reference/configuration/default-ports).
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:68]: Add 18018 here

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/azure.md [hunk 11]
```diff
@@ -77,9 +77,9 @@ You may choose not to create a new security group and instead use an existing on
...
+* View the YugabyteDB Anywhere UI (port `tcp:80`).
 
 If you are using your own custom Virtual Private Cloud (VPC), as it is typically the case for a self-managed configuration, the following additional TCP ports must be accessible: 7000, 7100, 9000, 9100, 11000, 12000, 9300, 9042, 5433, and 6379. For more information on ports used by YugabyteDB, refer to [Default ports](../../../../reference/configuration/default-ports).
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/azure.md:84]: Add 18018 here

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/azure.md [hunk 12]
```diff
@@ -90,7 +90,7 @@ To create a security group that enables these artifacts, navigate to **Network S
 * In the Name field, enter yugaware-sg or any other name.
 * Edit inbound security rules, as follows:
   * Add the appropriate IP addresses to the **Source IP ranges** field. To allow access from any machine, enter 0.0.0.0/0, but note that this is not very secure.
-  * Add the ports 22, 8800, and 80 to the **Port Range** field. The protocol selected must be TCP. For a self-managed configuration, also add 7000, 7100, 9000, 9100, 11000, 12000, 9300, 9042, 5433, and 6379 to the **Port Range** field.
+  * Add the ports 22, 8800, 80, and 18018 to the **Port Range** field. The protocol selected must be TCP. For a self-managed configuration, also add 7000, 7100, 9000, 9100, 11000, 12000, 9300, 9042, 5433, and 6379 to the **Port Range** field.
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/azure.md:93]: Rewrite this to 

Add ports 22, 8800, and 80 to the **Port Range** field. The protocol selected must be TCP. For a self-managed configuration, also add 7000, 7100, 9000, 9100, 18018, 11000, 12000, 9300, 9042, 5433, and 6379 to the **Port Range** field.

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md [hunk 13]
```diff
@@ -98,7 +98,7 @@ For more information, see [Creating and managing service accounts](https://cloud
...
 - View the YugabyteDB Anywhere UI (port `tcp:80`).
 
 If you are using your own Virtual Private Cloud (VPC) as a self-managed configuration, the following additional TCP ports must be accessible: 7000, 7100, 9000, 9100, 11000, 12000, 13000, 9300, 9042, 5433, 6379, 54422. For more information, see [Default ports](../../../../reference/configuration/default-ports).
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md:104]: Rewrite it as:

If you are using your own Virtual Private Cloud (VPC) as a self-managed configuration, the following additional TCP ports must be accessible: 7000, 7100, 9000, 9100, 18018, 11000, 12000, 13000, 9300, 9042, 5433, 6379, 54422. For more information, see [Default ports](../../../../reference/configuration/default-ports).

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md [hunk 14]
```diff
@@ -109,7 +109,7 @@ Next, you need to create a firewall entry, as follows:
 - Create firewall rules by following instructions provided in [Using firewall rules](https://cloud.google.com/vpc/docs/using-firewalls) in the GCP documentation. When creating the rules:
   - Add a tag `yugabyte-server` to the **Target tags** field.
   - Add the appropriate IP addresses to the **Source IP ranges** field.
-  - Enter a comma-delimited list of TCP ports 22, 8800, 80 to the **Protocol and ports** field. If required, also add TCP ports for a self-managed configuration.
+  - Enter a comma-delimited list of TCP ports 22, 8800, 80, and 18018 to the **Protocol and ports** field. If required, also add TCP ports for a self-managed configuration.
```
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md:112]: 18018 is not needed here

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md [hunk 15]
```diff
@@ -62,7 +62,7 @@ type: docs
 In order to access YugabyteDB Anywhere from outside the AWS environment, you would need to enable access by assigning an appropriate security group to the server hosting YugabyteDB Anywhere. At a minimum, you need to be able to do the following:
 
 - Access the YugabyteDB Anywhere instance over SSH (port `tcp:22`).
-- Check, manage, and upgrade YugabyteDB Anywhere (port `tcp:8800`).
+- Check, manage, and upgrade YugabyteDB Anywhere (port `tcp:8800` and `tcp:18018`).
```
Comment (vpatibandla-yb) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:65]: 18018 is not needed here

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md [hunk 16]
```diff
@@ -62,7 +62,7 @@ type: docs
...
 - View the YugabyteDB Anywhere UI (port `tcp:80`).
 
 If you are using your own Virtual Private Cloud (VPC) as a self-managed configuration, the following additional TCP ports must be accessible: 7000, 7100, 9000, 9100, 11000, 12000, 13000, 9300, 9042, 5433, 6379, 54422. For more information on ports used by YugabyteDB, refer to [Default ports](../../../../reference/configuration/default-ports).
```
Comment (vpatibandla-yb) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:68]: Add 18018 to the list of ports here

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md [hunk 17]
```diff
@@ -75,7 +75,7 @@ To create a security group that enables these, use the Amazon console to navigat
 
 - Add the appropriate IP addresses to the **Source** field. To allow access from any computer, add `0.0.0.0/0` but note that this is not very secure.
 
-- Add the ports 22, 8800, and 80 to the **Port Range** field. The **Protocol** selected must be TCP.
+- Add the ports 22, 8800, 80, and 18018 to the **Port Range** field. The **Protocol** selected must be TCP.
```
Comment (vpatibandla-yb) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:78]: 18018 is not needed here

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/azure.md [hunk 18]
```diff
@@ -77,9 +77,9 @@ You may choose not to create a new security group and instead use an existing on
 
 To access the YugabyteDB Anywhere from outside the Azure environment, you would need to enable access by assigning an appropriate network security group to the YugabyteDB Anywhere machine. At a minimum, you need to be able to do the following:
 
-* Access the YugabyteDB Anywhere instance over SSH (port tcp:22).
-* Check, manage, and upgrade YugabyteDB Anywhere (port tcp:8800).
-* View the YugabyteDB Anywhere UI (port tcp:80).
+* Access the YugabyteDB Anywhere instance over SSH (port `tcp:22`).
+* Check, manage, and upgrade YugabyteDB Anywhere (port `tcp:8800` and `tcp:18018`).
```
Comment (vpatibandla-yb) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/azure.md:81]: 18018 is not needed here

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/azure.md [hunk 19]
```diff
@@ -77,9 +77,9 @@ You may choose not to create a new security group and instead use an existing on
...
+* View the YugabyteDB Anywhere UI (port `tcp:80`).
 
 If you are using your own custom Virtual Private Cloud (VPC), as it is typically the case for a self-managed configuration, the following additional TCP ports must be accessible: 7000, 7100, 9000, 9100, 11000, 12000, 9300, 9042, 5433, and 6379. For more information on ports used by YugabyteDB, refer to [Default ports](../../../../reference/configuration/default-ports).
```
Comment (vpatibandla-yb) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/azure.md:84]: Add 18018 to the list of ports in custom configuration

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/azure.md [hunk 20]
```diff
@@ -90,7 +90,7 @@ To create a security group that enables these artifacts, navigate to **Network S
 * In the Name field, enter yugaware-sg or any other name.
 * Edit inbound security rules, as follows:
   * Add the appropriate IP addresses to the **Source IP ranges** field. To allow access from any machine, enter 0.0.0.0/0, but note that this is not very secure.
-  * Add the ports 22, 8800, and 80 to the **Port Range** field. The protocol selected must be TCP. For a self-managed configuration, also add 7000, 7100, 9000, 9100, 11000, 12000, 9300, 9042, 5433, and 6379 to the **Port Range** field.
+  * Add the ports 22, 8800, 80, and 18018 to the **Port Range** field. The protocol selected must be TCP. For a self-managed configuration, also add 7000, 7100, 9000, 9100, 11000, 12000, 9300, 9042, 5433, and 6379 to the **Port Range** field.
```
Comment (vpatibandla-yb) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/azure.md:93]: Rewrite it as:

Add the ports 22, 8800, and 80 to the **Port Range** field. The protocol selected must be TCP. For a self-managed configuration, also add 7000, 7100, 9000, 9100, 18018, 11000, 12000, 9300, 9042, 5433, and 6379 to the **Port Range** field.

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md [hunk 21]
```diff
@@ -98,7 +98,7 @@ For more information, see [Creating and managing service accounts](https://cloud
 In order to access YugabyteDB Anywhere from outside the GCP environment, you have to enable firewall rules. At a minimum, you need to be able to do the following:
 
 - Access the YugabyteDB Anywhere instance over SSH (port `tcp:22`).
-- Check, manage, and upgrade YugabyteDB Anywhere (port `tcp:8800`).
+- Check, manage, and upgrade YugabyteDB Anywhere (port `tcp:8800` and `tcp:18018`).
 - View the YugabyteDB Anywhere UI (port `tcp:80`).
 
 If you are using your own Virtual Private Cloud (VPC) as a self-managed configuration, the following additional TCP ports must be accessible: 7000, 7100, 9000, 9100, 11000, 12000, 13000, 9300, 9042, 5433, 6379, 54422. For more information, see [Default ports](../../../../reference/configuration/default-ports).
```
Comment (vpatibandla-yb) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md:104]: Add 18018 to the list of ports specified in the self-managed configuration

**Review by vpatibandla-yb — COMMENTED** [review]
Added the comprehensive list of places where 18018 port has to be added or is not needed

**Review by vpatibandla-yb — APPROVED** [review]

**Pull request merged**